src/peergos/server/tests/simulation/Simulator.java

```diff
@@ -461,8 +526,13 @@ public void run() {
             System.out.println("iOp=" + iOp);
             Simulation simulation = getNextSimulation.get();
             run(simulation);
+            /*
```

Comment:
commented  code to be removed

Reply:
2 modes, verify after each op, or verify after all ops. the former useful when debugging, later is what it was originally

Reply:
Oh I see. Leo free to add a flag to do this too :)

You still need to check everything at the end in case of regressions that affect fonts/folders other than the path directly referenced in the simulation op.